index.d.ts

```diff
@@ -8,50 +8,64 @@ declare class AbortErrorClass extends Error {
 }
 
 declare namespace pThrottle {
-	type ThrottledFunction<Arguments extends unknown[], Return> = ((
-		...arguments: Arguments
-	) => Promise<Return>) & {
+	type ThrottledFunction<FunctionType extends (...args: any) => any> = ((
+		...arguments: Parameters<FunctionType>
+	) => ReturnType<FunctionType>) & {
 		/**
 		Abort pending executions. All unresolved promises are rejected with a `pThrottle.AbortError` error.
 		*/
 		abort(): void;
 	};
 
+	interface Options {
+		/**
+		Maximum number of calls within an `interval`.
```

Comment:
Can you mention that the options are required? In the readme too.

Reply:
What's the cleanest way to do that?

Reply:
Probably as a note on line 45:

> Both the `limit` and `interval` options must be specified.